ui-lib/src/main/java/co/electriccoin/zcash/ui/common/provider/LocalAddressBookStorageProvider.kt

```diff
@@ -0,0 +1,44 @@
+package co.electriccoin.zcash.ui.common.provider
+
+import android.content.Context
+import co.electriccoin.zcash.spackle.Twig
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.FileOutputStream
+import java.io.IOException
+
+interface LocalAddressBookStorageProvider {
+    fun openStorageInputStream(): FileInputStream?
+
+    fun openStorageOutputStream(): FileOutputStream?
+}
+
+class LocalAddressBookStorageProviderImpl(
+    private val context: Context
+) : LocalAddressBookStorageProvider {
+    override fun openStorageInputStream(): FileInputStream? {
+        return try {
+            context.openFileInput(LOCAL_ADDRESS_BOOK_FILE_NAME)
```

Comment:
We agreed that we'd instead use `no_backup` and the app's subfolder for the new `address_book` file storage as we do for databases, block files, or Tor instances.